custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnTopBar.kt

```diff
@@ -0,0 +1,30 @@
+package jces1209.vu.action
+
+import com.atlassian.performance.tools.jiraactions.api.*
+import com.atlassian.performance.tools.jiraactions.api.action.Action
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import jces1209.vu.page.bars.topBar.AbstractTopBar
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+
+/**
+ * Works for both Cloud and Data Center.
+ */
+class WorkOnTopBar(
+    private val topBar: AbstractTopBar,
```

Comment:
It must be compilation error - there is no AbstractTopBar - it's just `TopBar` already